visual-js/visual-nightwatch/src/nightwatch/commands/sauceVisualCheck.ts

```diff
@@ -132,6 +132,10 @@ class SauceVisualCheck implements NightwatchCustomCommandsModel {
       ReturnType<typeof api.createSnapshotFromWebDriver>
     > | null = null;
 
+    const clipElement = options.clipSelector
```

Comment:
I would probably rename this to that:
```suggestion
    const clipElementFromClipSelector = options.clipSelector
```

It will help gain clarity on line 151:

From
```
clipElement: (await options.clipElement?.getId()) ?? clipElement,
```

to
```
clipElement: (await options.clipElement?.getId()) ?? clipElementFromClipSelector,
```

Otherwise you are in a situation where `option.clipElement` and `clipElement` are not as similar as naming could suggest.